### ISSUE TITLE
Videojs 5.0 API documentation missing argument names in some places

### ISSUE THREAD

**Issue by Soviut** [videojs/video.js#2711]
On the http://docs.videojs.com/docs/api/video.html page several methods have the wrong names for arguments, for example:

http://docs.videojs.com/docs/api/video.html#MethodsmergeOptions
http://docs.videojs.com/docs/api/video.html#Methodsplugin

This is because their params definitions in the jsdoc block are missing a name before their description and it's using the first word of the description as a name instead.

https://github.com/videojs/video.js/blob/master/src/js/video.js#L260

```javascript
 * @param {Object} The options object whose values will be overriden
 * @param {Object} The options object with values to override the first
 * @param {Object} Any number of additional options objects
```

When it should be something like:
```javascript
 * @param {Object} first     The options object whose values will be overriden
 * @param {Object} second    The options object with values to override the first
 * @param {Object} etc       Any number of additional options objects
```

**Comment by gkatsev** [comment]
Would you be willing to create a PR that adds in the missing names?

**Comment by Soviut** [comment]
Yes, I was going to but was having issues with the build failing. Do I actually need to run anything to update the docs?

**Comment by Soviut** [comment]
I've created the PR https://github.com/videojs/video.js/pull/2714 and it passed the TravisCI

**Comment by Soviut** [comment]
@gkatsev The build seems to be failing due to something unrelated

https://travis-ci.org/pam/video.js/builds/86235590
```
IE 9.0.0 (Windows Vista 0.0.0) ERROR: 'VIDEOJS:', 'ERROR:', {description: '[object Event]', message: '[object Event]', statusCode: 0}, {body: undefined, headers: {}, statusCode: 0, method: undefined, url: undefined, rawRequest: undefined}
```

Is there anything I can do?

**Comment by gkatsev** [comment]
Nope, I'll investigate the test failure.

**Comment by Soviut** [comment]
I've realized there are more missing names in the docs that I could fix.

**Comment by hartman** [comment]
:+1:

**Comment by gkatsev** [comment]
Fixed by #2714.
